# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-jemalloc-pagesize.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Arctic-Fox-45.3/memory/mozjemalloc/jemalloc.c.vanilla	2025-04-24 16:13:52.499049354 +0200
+++ Arctic-Fox-45.3/memory/mozjemalloc/jemalloc.c	2025-04-24 16:12:35.792234157 +0200
@@ -1050,8 +1050,8 @@
 #if (defined(SOLARIS) || defined(__FreeBSD__)) && \
     (defined(__sparc) || defined(__sparcv9) || defined(__ia64))
 #define pagesize_2pow			((size_t) 13)
-#elif defined(__powerpc64__) || defined(__aarch64__)
-#define pagesize_2pow			((size_t) 16)
+#elif defined(__powerpc64__) || defined(__aarch64__) || defined(__alpha__) || defined(__ia64__ )
+#define pagesize_2pow			((size_t) 14)
 #else
 #define pagesize_2pow			((size_t) 12)
 #endif
